Cubical/Categories/Category/Base.agda

```diff
@@ -0,0 +1,101 @@
+{-# OPTIONS --cubical --no-import-sorts --safe #-}
+
+
+module Cubical.Categories.Category.Base where
+
+open import Cubical.Core.Glue
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.HLevels
+open import Cubical.Foundations.Equiv
+
+private
+  variable
+    ℓ ℓ' : Level
+
+-- Precategories
+
+record Precategory ℓ ℓ' : Type (ℓ-suc (ℓ-max ℓ ℓ')) where
+  -- no-eta-equality ; NOTE: need eta equality for `opop`
+  field
+    ob : Type ℓ
+    Hom[_,_] : ob → ob → Type ℓ'
+    id : ∀ x → Hom[ x , x ]
+    _⋆_ : ∀ {x y z} (f : Hom[ x , y ]) (g : Hom[ y , z ]) → Hom[ x , z ]
+    -- TODO: change these to implicit argument
```

Comment:
Change which to implicit?

Reply:
The morphism can usually be inferred, so I was planning on changing the `f` to implicit but hadn't gotten around to it.